src/components/Dashboard.js

```diff
@@ -62,8 +63,25 @@ const Dashboard = () => {
               </div>
             }
           />
+          <Route
+            path="MarketPlace/*"
+            element={
+              <div>
+                <h1>Free Market</h1>
+                <MarketPlaceInputs />
+              </div>
+            }
+          />
```

Comment:
I'm a bit confused seeing lines 57-65 and then lines 66-74

Reply:
Kevin and I are meeting up for 3:30ish today to work on the component, I will make sure this is fixed :) You're welcome to join us if you'd like!